File: src/main/kotlin/application/controller/dialogs/rule/misc/TurmitesDialog.java
package application.controller.dialogs.rule.misc;

import application.controller.dialogs.rule.RuleWidget;
import application.model.rules.misc.turmites.Turmites;

public class TurmitesDialog extends RuleWidget {
    public TurmitesDialog() {
        super();

        ruleFamily = new Turmites();
    }

    @Override
    public void updateRule(String rulestring) {
        ruleFamily.setRulestring(rulestring);
    }
}

File: src/main/kotlin/application/controller/dialogs/rule/misc/TurmitesDialog.kt
package application.controller.dialogs.rule.misc

import application.controller.dialogs.rule.RuleWidget
import application.model.rules.misc.turmites.Turmites

class TurmitesDialog : RuleWidget() {
    override fun updateRule(rulestring: String?) {
        ruleFamily!!.rulestring = rulestring
    }

    init {
        ruleFamily = Turmites()
    }
}